zap/src/main/java/org/zaproxy/zap/extension/api/ContextAPI.java

```diff
@@ -349,6 +346,17 @@ public ApiResponse handleApiAction(String name, JSONObject params) throws ApiExc
         return ApiResponseElement.OK;
     }
 
+    private void handleTech(String[] techNames, Consumer<Tech> handler) throws ApiException {
+        for (String techName : techNames) {
+            Tech t = getTech(techName);
+            if (t != null) {
```

Comment:
Should not need this check.

Reply:
I’d have to test over again. IIRC it prevents an NPE when the user input fails to match an actual Tech.

Reply:
The method `getTech(String)` throws if the tech does not exist.